cmd/kubeadm/test/cmd/token_test.go

```diff
@@ -28,15 +29,26 @@ const (
 	TokenExpectedRegex = "^\\S{6}\\.\\S{16}\n$"
 )
 
-var kubeadmPath = flag.String("kubeadm-path", filepath.Join(os.Getenv("KUBE_ROOT"), "cluster/kubeadm.sh"), "Location of kubeadm")
+var kubeadmPathFlag = flag.String("kubeadm-path", filepath.Join(os.Getenv("KUBE_ROOT"), "cluster/kubeadm.sh"), "Location of kubeadm")
```

Comment:
`filepath.Join(os.Getenv("KUBE_ROOT"), "cluster", "kubeadm.sh")` to use platform-specific separator.